spec/javascripts/health-care/health-care.spec.js

```diff
@@ -139,7 +139,7 @@ describe("Health Care Form", function() {
   beforeEach(function(done){
     var fixtureFrame = document.createElement('iframe');
     fixtureFrame.id = "fixture-frame";
-    fixtureFrame.src = '/health-care/form.html';
+    fixtureFrame.src = '/health-care/form/index.html';
```

Comment:
This is correct. The automatic creation of an index.html from form.html occurs cause @emilyville added a jekyll config change in `_config.yml`.